source/_assets/sass/_documentation.scss

```diff
@@ -1 +1,4 @@
-// Add your custom styles here
+img.gh-emoji {
```

Comment:
The style doesn't seem to be "applied" on the [preview](https://deploy-preview-19--geocoder-php.netlify.com/docs/#local) and thus the flags are too big IMHO but maybe it's an just and issue with the preview.

Reply:
Strange, it's working on my dev. Will have another look tomorrow.

Reply:
![image](https://user-images.githubusercontent.com/50683531/63815846-6ee15e00-c979-11e9-8f6a-9e12cf0157ad.png)

Reply:
For some reason, it's not in the CSS that's being compiled 🤔 
https://deploy-preview-19--geocoder-php.netlify.com/assets/build/css/main.css

Reply:
Ok, fixed now :) Needed to add it to the purgecss whitelist.

Reply:
Fixed indeed 👍